### DISCUSSION TITLE
Question about "Explaining Models by Propagating Shapley Values"

### DISCUSSION THREAD

**Discussion by valleval97** [shap/shap#2999] [Q&A]
Hello, I would like to know, if it were possible, if the results like the one presented in Figure 3 of the mentioned article would be different if the sample suffers a considerable variation. I mean: Could be the summary of DeepSHAP attributions presented in that figure very sensitive to changes in the distribution of the original random background sample? Or maybe the results would be more or less stable as the model doesn't change as its parameters are constant? Thank you very much for your contribution.

**Comment by jainraj** [comment]
My two cents: SHAP values depend on the background dataset. So, if there are changes in the dataset, there will be changes in the learnt SHAP values. If we have large enough background dataset, WLLN assures us that we would be close to the true mean [1]. Hence, the attributions in the figure wouldn't be "very sensitive" as long as the background had enough samples. 

[1] - WLLN assumes the expectation to exist and in this case, they do, so we are in good footing here.
